packages/flutter/lib/src/material/floating_action_button_theme.dart

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+/// Defines default property values for descendant [FloatingActionButton]
+/// widgets.
+///
+/// Descendant widgets obtain the current [FloatingActionButtonThemeData] object
+/// using `FloatingActionButtonThemeData.of(context)`. Instances of
+/// [FloatingActionButtonThemeData] can be customized with
+/// [FloatingActionButtonThemeData.copyWith].
+///
+/// Typically a [FloatingActionButtonThemeData] is specified as part of the
+/// overall [Theme] with [ThemeData.floatingActionButtonTheme].
+///
+/// All [FloatingActionButtonThemeData] properties are `null` by default.
+/// When null, the [FloatingActionButton] will use the values from [ThemeData]
+/// if they exist, otherwise it will provide its own defaults.
+///
+/// See also:
+///
+///  * [ThemeData], which describes the overall theme information for the
+///    application.
+class FloatingActionButtonThemeData extends Diagnosticable {
+  /// Creates a theme that can be used for
+  /// [ThemeData.floatingActionButtonTheme].
+  const FloatingActionButtonThemeData({
+    this.backgroundColor,
+    this.foregroundColor,
+    this.elevation,
+    this.highlightElevation,
+    this.shape,
+  });
+
+  /// Color to be used for the unselected, enabled floating action buttons's
```

Comment:
Probably good place to use brackets:

`[FloatingActionButton]s`

here and below